sycl/CMakeLists.txt

```diff
@@ -218,6 +222,14 @@ add_custom_target( sycl-toolchain
   COMMENT "Building SYCL compiler toolchain..."
 )
 
+if (SYCL_ENABLE_XPTI_TRACING)
+  if (MSVC AND CMAKE_BUILD_TYPE MATCHES "Debug")
+    add_dependencies( sycl-toolchain xptid)
+  else()
+    add_dependencies( sycl-toolchain xpti)
```

Comment:
I think xpti should be always added if SYCL_ENABLE_XPTI_TRACING == true.

Reply:
Okay, reverted this back to how it was earlier.